components/VoteList/useVoteListItem.tsx

```diff
@@ -0,0 +1,326 @@
+import { green, grey100, isEarlyVote, red500 } from "constant";
+import {
+  decodeHexString,
+  formatVoteStringWithPrecision,
+  getPrecisionForIdentifier,
+} from "helpers";
+import { config } from "helpers/config";
+import {
+  useAccountDetails,
+  useAssertionClaim,
+  useDelegationContext,
+  useVotesContext,
+  useWalletContext,
+} from "hooks";
+import Link from "next/link";
+import Across from "public/assets/icons/across.svg";
+import OSnap from "public/assets/icons/osnap.svg";
+import Polymarket from "public/assets/icons/polymarket.svg";
+import UMAGovernance from "public/assets/icons/uma-governance.svg";
+import UMA from "public/assets/icons/uma.svg";
+import { CSSProperties, useEffect, useState } from "react";
+import removeMarkdown from "remove-markdown";
+import { DropdownItemT } from "types";
+import { VoteListItemProps } from "./shared.types";
+
+export function useVoteListItem({
```

Comment:
all of the state that lived in the `VoteListItem` has been copied here and left unchanged. This is now also used by the `VoteTableRow` component and the behavior is unchanged. the only thing that is removed is some logic for calculating table cell widths which are now redundant because we use an automatic table layout